File: CS/Factorial.cs
using System;

class Factorial
{
	public static void Main()
	{
		int n, f=1;
		
		Console.Write("Enter a number: ");
		n = Convert.ToInt32(Console.ReadLine());
		
		for(int i=n; i>=1; i--)
		{
			f = f * i;
		}
		
		Console.WriteLine("The factorial of {0} is {1}.", n, f);
	}
}

/*
	OOP
*/

File: Java/Factorial.java
import java.util.Scanner;
class Factorial {
	public static void main(String[] args) {
		int n, f=1;
		Scanner scan = new Scanner(System.in);
		System.out.print("Enter a number: ");
		n = scan.nextInt();
		for(int i=n; i>=1; i--) {
			f = f * i;
		}
		System.out.println("The factorial of " + n + " is " + f + ".");
	}
}